<meta name="robots" content="noindex" />
<pre>diff -up kdepim-3.5.10/libkcal/listbase.h.me kdepim-3.5.10/libkcal/listbase.h
--- kdepim-3.5.10/libkcal/listbase.h.me	2012-02-08 15:18:18.076644288 +0100
+++ kdepim-3.5.10/libkcal/listbase.h	2012-02-08 15:19:20.577257147 +0100
@@ -67,7 +67,7 @@ class ListBase : public QValueList&lt;T *&gt;
 
     bool removeRef( T *t )
     {
-      QValueListIterator&lt;T *&gt; it = find( t );
+      QValueListIterator&lt;T *&gt; it = this-&gt;find( t );
       if ( it == QValueList&lt;T*&gt;::end() ) {
         return false;
       } else {
</pre>
